test/test_masked.py

```diff
@@ -126,28 +126,35 @@ def apply_masked_reduction_along_dim(op, input, *args, **kwargs):
     return output
 
 
-def apply_masked_normalization_along_dim(op, x, dim, dtype=None, mask=None):
```

Comment:
Based on this change, this PR might need rebasing now that https://github.com/pytorch/pytorch/pull/68584 is merged?

Reply:
Rebase done.